Tests/src/PublicAPITests/ComponentTests/ComponentTests.cpp

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
+#include "../SpaceSystemTestHelpers.h"
```

Comment:
Should these paths not be relative to the project root like the rest? Or is there a reason for the file relative path?

Reply:
![image](https://github.com/user-attachments/assets/03a6b710-8f37-4ba8-8cad-e6a7ebd3f458)
Component tests is in a nested folder and we dont have an include dir setup at the root

Reply:
OK fair enough. Thought that might be your answer, but just wanted to check. It's not something I feel strongly about anyway.